blueocean-personalization/src/main/js/redux/FavoritesActions.jsx

```diff
@@ -142,6 +144,7 @@ export const actions = {
     generateData(request, actionType, optional) {
         const { url, fetchOptions } = request;
         return (dispatch) => Fetch.fetchJSON(url, { fetchOptions })
+            .then(data => augmenter.augmentCapabilities(data))
```

Comment:
@tfennelly is this similar to the idea/ideal you had for capabilities and adding them to results?